app/views/landing/index.html.erb

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
```

Comment:
The code between line 2 and 9 is already in the layout.html.haml file. No need for it here.

Reply:
Research what `yield` in the layout file is about